pkg/image/apis/image/helper.go

```diff
@@ -523,6 +539,44 @@ func ImageWithMetadata(image *Image) error {
 	return nil
 }
 
+// FetchImageSignature attemps to fetch the signature blob for the provided
+// image and mutates the image object with the signature content if the registry
+// has defined a known signature store.
+func FetchImageSignature(registry string, image *Image) error {
+	for name, sigStore := range knownSignatureStores {
+		if registry != name {
+			continue
+		}
+		// TODO: Should we increase the trust to signature stores by verifying the HTTPS certificates?
+		client := &http.Client{
+			Transport: &http.Transport{
+				TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
@mtrmac i think i'm going to use the containers/image transport to get signatures (with a little bit of tweaking and wrapping with timeout)